File: 프로그래머스/lv0/120894. 영어가 싫어요/영어가 싫어요.py
def solution(numbers):
    numbers = numbers.replace("zero","0").replace("one","1").replace("two","2").replace("three","3").replace("four","4").replace("five","5").replace("six","6").replace("seven","7").replace("eight","8").replace("nine","9")
    
    
    return int(numbers)

File: 프로그래머스/0/120894. 영어가 싫어요/영어가 싫어요.java
class Solution {
    public long solution(String numbers) {
        String numStr = numbers.replace("zero","0")
            .replaceAll("one","1")
            .replaceAll("two","2")
            .replaceAll("three","3")
            .replaceAll("four","4")
            .replaceAll("five","5")
            .replaceAll("six","6")
            .replaceAll("seven","7")
            .replaceAll("eight","8")
            .replaceAll("nine","9");
        
        return Long.parseLong(numStr);
    }
}